pkg/lib/indexer/indexer.go

```diff
@@ -696,3 +776,124 @@ func (i ImageIndexer) DeprecateFromIndex(request DeprecateFromIndexRequest) erro
 
 	return nil
 }
+
+func ensureCSVFields(bundleDir string, bundleInfo bundleDirPrefix) error {
+	dirContent, err := ioutil.ReadDir(bundleDir)
+	if err != nil {
+		return fmt.Errorf("error reading bundle directory %s, %v", bundleDir, err)
+	}
+
+	var foundCSV bool
```

Comment:
suggestion: instead of using this bool we could just break when we find a CSV? Then the rest of the code can be outside the for loop and potentially in its own function.

There shouldn't be a case where there is more than one CSV in a bundle that is to be exported from an index. It's possible if the user uses entirely their own tooling, but I'm not sure we need to worry about it here.